Capri.Web.ViewModels/Proposal/ProposalRegistration.cs

```diff
@@ -9,6 +9,7 @@ public class ProposalRegistration
         public string TopicPolish { get; set; }
         public string TopicEnglish { get; set; }
         public string Description { get; set; }
+        public string OutputData { get; set; }
```

Comment:
Can you explain what's the purpose of that property?

Reply:
I've added this property, because I forgot to do this before. This is for "Dane wyjściowe" section in the diploma card.